deploy/kubernetes/helm/che/values.yaml

```diff
@@ -32,6 +32,8 @@ global:
   #  This value can be passed if custom Oidc provider is used, and there is no need to deploy keycloak in multiuser mode
   #  default (if empty) is true
   #cheDedicatedKeycloak: false
+  ingressClass: "ingress"
```

Comment:
The previously hardcoded value was "nginx". Now it would be pointing to this value which is "ingress". Shouldn't it be "nginx"?